thewalrus/tests/test_entanglement.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Entanglement tests"""
+# pylint: disable=no-self-use, assignment-from-no-return
```

Comment:
Yes, you can do that. If they're needed, pyfactor will complain in the CI. In many cases they can be addressed rather than disabled, and we should only do the latter as a last resort.